### DISCUSSION TITLE
Friday night funkin kade engine 1.6.1 how to add week and free play songs

### DISCUSSION THREAD

**Discussion by Reyororo** [KadeArchive/Kade-Engine#1630] [Q&A] [answered]
Please tell me how to add a week and add a new free play song

**Comment by raf13lol** [comment, marked as answer]
To Add A Free Play Song Edit The freeplaySonglist.txt file. To Add A Week Edit The StoryMenuState.hx And Add A Image With It.

**Reply by Reyororo** [reply]
> フリープレイソングを追加するにはfreeplaySonglist.txtファイルを編集します。1週間を追加するには、StoryMenuState.hxを編集し、それを使用して画像を追加します。

Please tell me the compilation procedure until you can set the song and play it.

**Reply by raf13lol** [reply]
you mean compile?

**Reply by Reyororo** [reply]
> you mean compile?

Maybe yes, I'm not very good at English and rely on Google Translate so I might make a mistake

**Reply by Reyororo** [reply]
Was self resolved

**Reply by greplol** [reply]
tranlation: Edit the freeplaySonglist.txt file to add freeplay songs. To add a week, edit StoryMenuState.hx and use it to add images.
